include/unifex/sender_concepts.hpp

```diff
@@ -144,6 +133,23 @@ namespace detail {
     using _unspecialized = void;
   };
 
+#ifndef _MSC_VER
+  template <typename S>
+  UNIFEX_CONCEPT_FRAGMENT(  //
+    _not_has_sender_traits, //
+      requires() (          //
+        typename (typename sender_traits<S>::_unspecialized)
+      ));
+  template <typename S>
+  UNIFEX_CONCEPT         //
+    _has_sender_traits = //
+      (!UNIFEX_FRAGMENT(detail::_not_has_sender_traits, S));
+#else
+  template <typename S>
+  inline constexpr bool _has_sender_traits =
+      !std::is_base_of_v<_no_sender_traits, sender_traits<S>>; 
```

Comment:
Do you know why this change was necessary for MSVC?
Maybe add a comment to that effect - at least indicating which version of MSVC was problematic so we know when to revisit.

Reply:
I don't know why it was necessary. I'll add a comment.